tools/packaging/common/istio-start.sh

```diff
@@ -116,11 +117,18 @@ ISTIO_AGENT_FLAGS=${ISTIO_AGENT_FLAGS:-}
 # Split ISTIO_AGENT_FLAGS by spaces.
 IFS=' ' read -r -a ISTIO_AGENT_FLAGS_ARRAY <<< "$ISTIO_AGENT_FLAGS"
 
-export PROXY_CONFIG=${PROXY_CONFIG:-"
+DEFAULT_PROXY_CONFIG="
 serviceCluster: $SVC
 controlPlaneAuthPolicy: ${CONTROL_PLANE_AUTH_POLICY}
-discoveryAddress: ${PILOT_ADDRESS}
-"}
+"
+if [ -n "${CUSTOM_PILOT_ADDRESS}" ]; then
+  PROXY_CONFIG="$PROXY_CONFIG
+  discoveryAddress: ${CUSTOM_PILOT_ADDRESS}
```

Comment:
```suggestion
discoveryAddress: ${CUSTOM_PILOT_ADDRESS}
```
spacing is suspicious here?